modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -136,6 +136,19 @@ type Handler struct {
 	// used for the requests and responses (in bytes).
 	MaxBufferSize int64 `json:"max_buffer_size,omitempty"`
 
+	// If true, the request body will not be copied to the backend.
+	// This is useful if you wish to send a request to a backend
+	// to have it decide whether the request should continue being
+	// handled; also known as forward auth.
+	// By default, this is false (disabled).
+	NoBody bool `json:"no_body,omitempty"`
+
+	// If set, overrides the request method on the upstream request.
+	// This is useful for a forward auth scenario, where the method
+	// should always be `GET`.
+	// By default, the incoming request's method is used.
+	OverrideMethod string `json:"override_method,omitempty"`
```

Comment:
Is this different from a method rewrite?

Reply:
Yes because `method` directive will affect the original request, not just the clone.